src/platform/model.py

```
@@ -28,58 +27,47 @@ def format(self) -> str:
-        rank_text = StyledString(rank, 'H', 44, font_color=text_color, padding_bottom=-6)
-        rating_text = StyledString(f"{rating}", 'H', 256, font_color=text_color, padding_bottom=44)
-
-        current_x, current_y = 144 + 32, 120 + 32
-        current_y = draw_text(img, pf_text, current_x + 48 + 18, current_y)
-        current_y = draw_text(img, handle_text, current_x, current_y)
-        current_y = draw_text(img, social_text, current_x, current_y)
-        current_y = draw_text(img, rank_text, current_x, current_y)
-        draw_text(img, rating_text, current_x - 10, current_y)
-
-        return img
+        pass
```

Ruff (0.8.2)

`CompetitivePlatform._get_contest_list` is an empty method in an abstract base class, but has no abstract decorator

(B027)